processor/transformprocessor/README.md

```diff
@@ -1,10 +1,11 @@
 # Transform Processor
 
 | Status                   |                       |
-| ------------------------ | --------------------- |
+|--------------------------|-----------------------|
 | Stability                | [alpha]               |
 | Supported pipeline types | traces, metrics, logs |
 | Distributions            | [contrib]             |
+| Warnings                 | [Many](#warnings)     |
```

Comment:
I like the name "caveat" more, although I am not a native English speaker. "Limitations" also sounds like a reasonable name for some of the ones we discussed at the SIG, but maybe it doesn't apply accurately to all of the issues discussed.

Reply:
I was purposely trying to avoid the word "caveats" because I thought it might not be good for non-native English speakers. I can switch to Caveats if everything agrees it's ok.

Reply:
I see! Well 'caveats' is understandable to me in that sense, but I don't want to be the representative of non-native speakers 😄